--- tin-1.4.3/src/init.c.orig	Thu Dec 16 16:12:45 1999
+++ tin-1.4.3/src/init.c	Thu Dec 16 16:13:14 1999
@@ -293,7 +293,7 @@
 	0,		/* col_title (initialised later) */
 	2,		/* word_h_display_marks */
 	TRUE,		/* word_highlight */
-	FALSE,		/* use_color */
+	TRUE,		/* use_color */
 #endif /* HAVE_COLOR */
 	TRUE,		/* add_posted_to_filter */
 	TRUE,		/* advertising */
@@ -637,7 +637,7 @@
 
 #ifdef HAVE_COLOR
 	preinit_colors();
-	use_color = FALSE;
+	use_color = TRUE;
 	word_highlight = TRUE;
 #endif /* HAVE_COLOR */
 
