diff -urN tin-1.5.9.org/src/init.c tin-1.5.9/src/init.c
--- tin-1.5.9.org/src/init.c	Fri Jul 13 12:42:12 2001
+++ tin-1.5.9/src/init.c	Mon Jul 23 23:40:22 2001
@@ -299,7 +299,7 @@
 	0,		/* col_title (initialised later) */
 	2,		/* word_h_display_marks */
 	TRUE,		/* word_highlight */
-	FALSE,		/* use_color */
+	TRUE,		/* use_color */
 #endif /* HAVE_COLOR */
 	TRUE,		/* add_posted_to_filter */
 	TRUE,		/* advertising */
@@ -600,7 +600,7 @@
 
 #ifdef HAVE_COLOR
 	preinit_colors();
-	use_color = FALSE;
+	use_color = TRUE;
 	word_highlight = TRUE;
 #endif /* HAVE_COLOR */
 
