diff -uNr tin-1.5.6.orig/src/Makefile.in tin-1.5.6/src/Makefile.in
--- tin-1.5.6.orig/src/Makefile.in	Thu Aug  3 15:41:43 2000
+++ tin-1.5.6/src/Makefile.in	Fri Aug 18 03:24:59 2000
@@ -50,7 +50,7 @@
 
 DEBUG	= @ENABLE_DEBUG@ -DDEBUG #-DDEBUG_NEWSRC -DPROFILE -DDEBUG_REFS
 CANLOCK = # -DUSE_CANLOCK # -DEVIL_INSIDE
-CPPOPTS	= @DEFS@ -I. -I../include -I$(top_builddir)/pcre -I$(top_srcdir)/pcre -DLOCALEDIR=\"$(prefix)/@DATADIRNAME@/locale\" -I$(INCDIR) $(DEBUG) $(CANLOCK) @CPPFLAGS@
+CPPOPTS = @DEFS@ -I/usr/include/ncurses -I. -I../include -DLOCALEDIR=\"$(prefix)/@DATADIRNAME@/\"
 CFLAGS	= @CFLAGS@
 
 LD	= $(CC)
@@ -58,7 +58,7 @@
 YACC	= @YACC@
 
 CANLIB	= # -L../libcanlock -lcanlock
-LIBS	= -L../pcre -lpcre $(CANLIB) @LIBS@ @INN_NNTPLIB@ @INTLDIR_MAKE@ @INTLLIBS@
+LIBS	= -lpcre $(CANLIB) @LIBS@ @INN_NNTPLIB@ @INTLDIR_MAKE@ @INTLLIBS@
 
 # Where do you want the binary & manual page installed?
 DESTDIR	= @DESTDIR@
@@ -178,7 +178,7 @@
 
 EXTRAOBJS = @EXTRAOBJS@
 
-EXTRALIBS = ../pcre/$(PRELIB)pcre.a @INTLDIR_MAKE@ @INTLLIBS@
+EXTRALIBS = @INTLDIR_MAKE@ @INTLLIBS@
 
 OFILES	= @ALLOCA@ $(EXTRAOBJS) \
 	$(OBJDIR)/active.o \
@@ -380,7 +380,7 @@
 
 cflow:
 	@$(ECHO) "Creating cflow for $(PROJECT)..."
-	@$(CFLOW) -I$(INCDIR) -I$(SRCDIR) -I../pcre $(CFILES) > cflow.$(PROJECT)
+	@$(CFLOW) -I$(INCDIR) -I$(SRCDIR) $(CFILES) > cflow.$(PROJECT)
 
 cscope:
 	@$(ECHO) "Creating cscope database $(PROJECT)..."
@@ -399,10 +399,6 @@
 	$(INCDIR)/nntplib.h \
 	$(INCDIR)/proto.h
 
-
-../pcre/$(PRELIB)pcre.a:
-	@$(ECHO) "Building Philip Hazel's Perl regular expressions library..."
-	@$(CD) ../pcre && $(MAKE) @cf_cv_makeflags@ $(PRELIB)pcre.a
 
 @INTLDIR_MAKE@@INTLLIBS@:
 @INTLDIR_MAKE@	@$(ECHO) "Building GNU gettext library..."
