--- tin-2.4.5/doc/tin.defaults~	2017-05-03 17:47:32.000000000 +0000
+++ tin-2.4.5/doc/tin.defaults	2021-08-14 07:50:47.703578241 +0000
@@ -55,7 +55,12 @@
 # idea to set mm_charset here for providing a default
 # in most western countries ISO-8859-1 might be the best value
 #
-#mm_charset=iso-8859-1
+mm_charset=ISO-8859-2
+
+# charset used for MIME (Content-Type) header in postings.
+
+mm_network_charset=ISO-8859-2
+
 
 # post_mime_encoding
 #
@@ -63,7 +68,7 @@
 # idea to set post_mime_encoding here for providing a default
 # in most western countries 8bit might be the best value
 #
-#post_mime_encoding=8bit
+post_mime_encoding=8bit
 
 # mail_mime_encoding
 #
@@ -155,3 +160,8 @@
 # to change this setting if your configuration differs
 #
 #subscriptionsfile=/usr/lib/news/subscriptions
+
+# Thread articles on 0=(nothing) 1=(Subject) 2=(References) 3=(Both)
+# 4=(Multipart Subject).
+thread_articles=3
+
