diff -urN tin-1.5.9.org/configure tin-1.5.9/configure
--- tin-1.5.9.org/configure	Mon Jul 23 14:31:37 2001
+++ tin-1.5.9/configure	Wed Jul 25 21:33:46 2001
@@ -797,6 +797,7 @@
 ALL_LINGUAS="de et en_GB"
 
 ### Checks for programs
+CFLAGS="$CFLAGS -I/usr/include/ncurses"
 save_CFLAGS="$CFLAGS"
 # Extract the first word of "gcc", so it can be a program name with args.
 set dummy gcc; ac_word=$2
diff -urN tin-1.5.9.org/configure.in tin-1.5.9/configure.in
--- tin-1.5.9.org/configure.in	Mon Jul 23 14:31:35 2001
+++ tin-1.5.9/configure.in	Wed Jul 25 21:34:14 2001
@@ -82,6 +82,7 @@
 ALL_LINGUAS="de et en_GB"
 
 ### Checks for programs
+CFLAGS="$CFLAGS -I/usr/include/ncurses"
 save_CFLAGS="$CFLAGS"
 AC_PROG_CC
 AC_PROG_GCC_TRADITIONAL
